addons/binding/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/config/MystromBridgeConfiguration.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mystrom.config;
```

Comment:
recent convention requires all classes not exposed should be put in a internal package: e.g. `...mystron.internal.config`